.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -21,7 +21,7 @@ body:
       label: Checklist
       description: A Checklist is provided below for you to track each of the subsequent tasks.
       options:
-        - label: Re-run analysis with `topostats process --core 1`.
+        - label: If TopoStats halts processing, re-run analysis with `topostats process --core 1`.
```

Comment:
As preparatory work for the #517 I moved the common options to be arguments to the `topostats` command rather than the sub-parsers.

Thus if you run as suggested you get an error as `cores` is not an argument to the `process` sub-processor...

```
❱ topostats process --cores 1
usage: topostats [-h] [-v] [-c CONFIG_FILE] [-s SUMMARY_CONFIG] [--matplotlibrc MATPLOTLIBRC] [-b BASE_DIR] [-o OUTPUT_DIR]
                 [-l LOG_LEVEL] [-j CORES] [-f FILE_EXT] [--channel CHANNEL]
                 {process,load,filter,grains,grainstats,disordered-tracing,nodestats,ordered-tracing,splining,summary,create-config,create-matplotlibrc}
                 ...
topostats: error: unrecognized arguments: --cores 1
```

Rather its an argument to `topostats`

```
❱ topostats --help
usage: topostats [-h] [-v] [-c CONFIG_FILE] [-s SUMMARY_CONFIG] [--matplotlibrc MATPLOTLIBRC] [-b BASE_DIR] [-o OUTPUT_DIR]
                 [-l LOG_LEVEL] [-j CORES] [-f FILE_EXT] [--channel CHANNEL]
                 {process,load,filter,grains,grainstats,disordered-tracing,nodestats,ordered-tracing,splining,summary,create-config,create-matplotlibrc}
                 ...

Run various programs relating to AFM data. Add the name of the program you wish to run.

options:
  -h, --help            show this help message and exit
  -v, --version         Report the current version of TopoStats that is installed
  -c CONFIG_FILE, --config-file CONFIG_FILE
                        Path to a YAML configuration file.
  -s SUMMARY_CONFIG, --summary-config SUMMARY_CONFIG
                        Path to a YAML configuration file for summary plots and statistics.
  --matplotlibrc MATPLOTLIBRC
                        Path to a matplotlibrc file.
  -b BASE_DIR, --base-dir BASE_DIR
                        Base directory to scan for images.
  -o OUTPUT_DIR, --output-dir OUTPUT_DIR
                        Output directory to write results to.
  -l LOG_LEVEL, --log-level LOG_LEVEL
                        Logging level to use, default is 'info' for verbose output use 'debug'.
  -j CORES, --cores CORES
                        Number of CPU cores to use when processing.
  -f FILE_EXT, --file-ext FILE_EXT
                        File extension to scan for.
  --channel CHANNEL     Channel to extract.

program:
  Available programs, listed below:

  {process,load,filter,grains,grainstats,disordered-tracing,nodestats,ordered-tracing,splining,summary,create-config,create-matplotlibrc}
    process             Process AFM images. Additional arguments over-ride defaults or those in the configuration file.
    load                Load and save all images as .topostats files for subsequent processing.
    filter              WIP DO NOT USE - Load and filter images, saving as .topostats files for subsequent processing.
    grains              WIP DO NOT USE - Load filtered images from '.topostats' files and detect grains.
    grainstats          WIP DO NOT USE - Load images with grains from '.topostats' files and calculate statistics.
    disordered-tracing  WIP DO NOT USE - Skeletonise and prune objects to disordered traces.
    nodestats           WIP DO NOT USE - Calculate node statistics and disentangle molecules.
    ordered-tracing     WIP DO NOT USE - Ordered traces of pruned skeletons.
    splining            WIP DO NOT USE - Splining of traced molecules to produce smooth curves.
    summary             Plotting and summary of TopoStats output statistics.
    create-config       Create a configuration file using the defaults.
    create-matplotlibrc
                        Create a Matplotlibrc parameters file using the defaults.
```

...and so the advice should be...

```suggestion
        - label: If TopoStats halts processing, re-run analysis with `topostats --cores 1 process`.
```